emulator/blockchain.go

```diff
@@ -530,6 +530,19 @@ func (h CadenceHook) Run(_ *zerolog.Event, level zerolog.Level, msg string) {
 	}
 }
 
+// `dummyEntropyProvider“ implements `environment.EntropyProvider`
+// which provides a source of entropy to fvm context (required for Cadence's randomness).
+type dummyEntropyProvider struct{}
+
+var dummySource = make([]byte, 32)
```

Comment:
This causes randomness on emulator to be:
 1. deterministic
 2. same for all txs/blocks
 
 Feel free to update this to be any behaviour you prefer.